x-pack/plugins/cases/server/client/utils.ts

```diff
@@ -456,13 +456,23 @@ export const getCaseToUpdate = (
     { id: queryCase.id, version: queryCase.version }
   );
 
+/**
```

Comment:
Is there an issue for this?

Reply:
Yes, https://github.com/elastic/kibana/issues/153726 section "Strict input validation"